Sync/StorageClient.swift

```diff
@@ -350,12 +351,22 @@ public class Sync15StorageClient {
         }
     }
 
+    lazy private var alamofire: Alamofire.Manager = {
+        var defaultHeaders = Alamofire.Manager.sharedInstance.session.configuration.HTTPAdditionalHeaders ?? [:]
+        defaultHeaders["User-Agent"] = userAgent
```

Comment:
Actually, better is to directly call `UserAgent.defaultUserAgent()` here, and get rid of the `userAgent` var altogether — otherwise we do the expensive UA computation eagerly when this class is loaded.